README.md

```diff
@@ -4,80 +4,107 @@
 * Java 11
 * Docker
 
-## Building
-```
+## Build Procedures
+
+### Build
+To build and test the application:
+
+```bash
 ./gradlew build
 ```
-Because the Docker images are built automatically, a docker daemon must be available.
+(NOTE: Because the Docker images are built automatically, a docker daemon must be available.)
 
+To run OWASP checks
 
-### Tests
-Tests are run automatically with `./gradlew build`. Even if the tests fail, the artifacts are built and can be run.
+```bash
+./gradlew dependencyCheckAnalyze --info
+```
 
-To run tests with quiet logs, add `-Pquiet`. This should not be used with parallel builds.
+The build can fail because the static analysis tool, Spotless, detects an issue. To correct
+most issues, ask Spotless to apply style and formatting changes:
 
-To run a single test suite, execute:
+```bash
+./gradlew spotlessApply
 ```
+For more information about spotless checks see
+[here](https://github.com/diffplug/spotless/tree/master/plugin-gradle#custom-rules).
+
+
+### Test
+* Tests are run automatically with `./gradlew build`.
+* Even if the tests fail, the artifacts are built and can be run.
+* To run tests with quiet logs, add `-Pquiet`. This should not be used with parallel builds.
+* To run a single test suite, execute:
+
+```bash
 ./gradlew module:test --test fullClassName
 ```
 
-#### Integration Tests
-The integration tests require that Docker is running.
+* Integration tests
+* The integration tests require that Docker is running.
+* To skip integration tests, add `-PskipITests`.
 
-To skip integration tests, add `-PskipITests`.
+----
 
-## Running
-1. Boot up an [Apache Cassandra](https://cassandra.apache.org/) instance using the default hostname and ports: `localhost:9042`.
+## Run Services Locally
```

Comment:
✏️ Running Services Locally

Reply:
I re-worded the section headings to be imperatives instead of gerunds.